front/style/layout/_buttons.scss

```diff
@@ -233,10 +233,10 @@ input {
 
 .btn--as-link {
     &--error {
-        color: $red!important;
+        color: $red;
         &:hover {
-            border-color: $red!important;;
-            color: $red!important;;
+            border-color: $red;;
```

Comment:
Why double `;` ? I think that was a mistake, can you remove this ?

Reply:
oO, I just revert ^^, it was like that before start this issue, I will fix it anyway